about.html

```diff
@@ -0,0 +1,142 @@
+
+</head>
+
+<body>
+    <section class="top-nav">
+        <div></div>
+        <div class="top-nav-content">
+            <img src="./items/facebook-icon.png" alt="Facebook icon">
+            <img src="./items/twitter-icon.png" alt="twiter icon">
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Logout</a>
+        </div>
+    </section>
+    <header>
+        <a href="index.html" class="logo">
+            <img src="./items/Logo.png" alt="this is the image of the logo">
+        </a>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        <div class="nav-lists">
+            <a href="./index.html" class="nav-link">Home</a>
+            <a href="" class="nav-link">Schedule</a>
+            <a href="" class="nav-link">Contact-Us</a>
+            <button>Get Ticket</button>
+
+        </div>
+    </header>
+    <section class="about-section">
+        <p class="about-section-firstParagraph">Hello!Music Fans</p>
+        <h1 class="about-heading">Muzik best for experience</h1>
+        <p class="about-section-secondParagraph">This is a platform for selling concert tickets and it is singer-three
+            to showcasing the ability of the global singers through providing the videos
+            of their lasts concerts and tour yes you can just go ahead and subscribe to the chanel for you to be updated.
+        </p>
+        <p class="about-third-paragraph">Please conact us per email for any futher questions on Muzik and Tickets</p>
+        <h2 class="about-email">concert@muzik.org</h2>
+        <hr>
+    </section>
+    <section class="tier-section">
+        <div class="tier-headding">
+            <h1> Grab your Tier here.! </h1>
+            <hr>
+        </div>
+        <div class="tier-cards">
+        <div class="tier-content">
+            <h2>Basic</h2>
+            <h3>$200</h3>
+            <p>1 Seat Available</p>
+            <p>Access To Wifi Router</p>
+            <p>Free Food & Coffee</p>
+            <button>Get Ticket</button>
+        </div>
+        <div class="tier-content2">
+            <h2>Standard</h2>
+            <h3>$250</h3>
+            <p class="seats">1 Seat Available</p>
+            <p class="seats">Access To Wifi Router</p>
+            <p class="seats">Free Food & Coffee</p>
+            <button>Get Ticket</button>
+        </div>
+        <div class="tier-content3">
+            <h2>Premium</h2>
+            <h3>$200</h3>
+            <p class="seats">2 Seat Available</p>
+            <p class="seats">Access To Wifi Router</p>
+            <p class="seats">Free Food & Coffee</p>
+            <button>Get Ticket</button>
+        </div>
+    </div>
+    </section>
+
+    <section class="past-events">
+
+        <div class="past-event-headings">
+            <h1>See The Past Concerts</h1>
```

Comment:
- [ ] This is a subtitle so it would be nice if you used a hx tag such as h2, h3 e.t.c instead of the `h1` tag which should be reserved fo the main title of the page, for semantic purposes